dotnetv3/KMS/DescribeKeyExample/DescribeKey.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+/// <summary>
+/// Retrieve information about an Amazon Key Management Service key. You can
+/// supply either the Key Id or the Key ARN to the DescribeKeyRequest KeyId
+/// property. The example was created using the AWS SDK for .NET version 3.7
+/// and .NET Core 5.0.
+/// </summary>
+namespace DescribeKeyExample
+{
+    // snippet-start:[KMS.dotnetv3.DescribeKeyExample]
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.KeyManagementService;
+    using Amazon.KeyManagementService.Model;
+
+    public class DescribeKey
+    {
+        public static async Task Main()
+        {
+            var keyId = "7c9eccc2-38cb-4c4f-9db3-766ee8dd3ad4";
+            var request = new DescribeKeyRequest
+            {
+                KeyId = keyId,
+            };
+
+            var client = new AmazonKeyManagementServiceClient();
+
+            var response = await client.DescribeKeyAsync(request);
+            var metadata = response.KeyMetadata;
+
+            Console.WriteLine($"{metadata.KeyId} created on: {metadata.CreationDate}");
+            Console.WriteLine($"State: {metadata.KeyState}");
+            Console.WriteLine($"{metadata.Description}");
+        }
+    }
+
```

Comment:
Remove this blank line.

Reply:
The linter calls for a blank line before a single line comment. All of the examples have a blank line before the closing snippet tag.